includes/components/class-attribute.php

```diff
@@ -400,6 +467,7 @@ public function register_attributes() {
 						'searchable'     => (bool) $attribute_object->hp_searchable,
 						'filterable'     => (bool) $attribute_object->hp_filterable,
 						'sortable'       => (bool) $attribute_object->hp_sortable,
+						'allow_unique'   => (bool) $attribute_object->hp_allow_unique,
```

Comment:
Please rename it to just "unique", similarly to other flags using a single adjective

Reply:
Let me know in tg why this implementation has the "unique"-related code, is this branch based on another task's branch where there's an option to enforce the attribute unique value?